libcontainer/specconv/spec_linux.go

```diff
@@ -1016,14 +1016,19 @@ func SetupSeccomp(config *specs.LinuxSeccomp) (*configs.Seccomp, error) {
 		return nil, nil
 	}
 
-	// We don't currently support seccomp flags.
-	if len(config.Flags) != 0 {
-		return nil, errors.New("seccomp flags are not yet supported by runc")
-	}
-
 	newConfig := new(configs.Seccomp)
 	newConfig.Syscalls = []*configs.Syscall{}
 
+	// We don't currently support all seccomp flags.
+	for _, flag := range config.Flags {
+		switch flag {
+		case "SECCOMP_FILTER_FLAG_LOG":
+			newConfig.Flags = append(newConfig.Flags, "SECCOMP_FILTER_FLAG_LOG")
+		default:
+			return nil, fmt.Errorf("seccomp flags %q not yet supported by runc", flag)
```

Comment:
Just curious (I don't remember this by now, it's been a few months since I changed this part) we need to have this logic here, instead of relying that when calling InitSeccomp(), if a not supported flag is used the error will be shown, because InitSeccomp() might run later in the chain and the error is not so clear/visible at that point (maybe the stdout was closed or something)?

If we can avoid to repeat it, it will be great. But IIRC something like that happened and it was best to do some checks here and more clear errors

Reply:
I am not sure, the check was done at this place so I kept the current structure